ttnn/cpp/ttnn/operations/full/full_pybind.cpp

```diff
@@ -0,0 +1,60 @@
+// SPDX-FileCopyrightText: © 2023 Tenstorrent Inc.
+//
+// SPDX-License-Identifier: Apache-2.0
+
+#include "full_pybind.hpp"
+
+#include <pybind11/pybind11.h>
+#include <pybind11/stl.h>
+
+#include "full.hpp"
+#include "pybind11/cast.h"
+#include "ttnn/cpp/pybind11/decorators.hpp"
+#include "ttnn/operations/full/device/full_device_operation.hpp"
+
+namespace ttnn::operations::full {
+void bind_full_operation(py::module& module) {
+    auto doc = fmt::format(
+        R"doc(
+        Creates a tensor of the specified shape and fills it with the specified scalar value.
+
+        Args:
+            shape (ttnn.Shape): The shape of the tensor.
+            fill_value (float): The value to fill the tensor with.
+            dtype (ttnn.DataType, optional): The data type of the tensor. Defaults to `None`.
+            layout (ttnn.Layout, optional): The layout of the tensor. Defaults to `None`.
+            device (ttnn.Device, optional): The device on which the tensor will be allocated. Defaults to `None`.
+            memory_config (ttnn.MemoryConfig, optional): The memory configuration of the tensor. Defaults to `None`.
+            output_tensor (ttnn.Tensor, optional): Preallocated output tensor. Defaults to `None`.
+
+        Note:
+            ROW_MAJOR_LAYOUT requires last dimension (shape[-1]) to be a multiple of 2 with dtype BFLOAT16 or UINT16.
+            TILE_LAYOUT requires width (shape[-1]) and height (shape[-2]) dimension to be multiple of 32.
+
+        Returns:
+            ttnn.Tensor: A filled tensor of specified shape and value.
+
+        Example:
+            >>> filled_tensor = ttnn.moreh_full(shape=[2, 2], fill_value=7.0, dtype=ttnn.bfloat16)
+            >>> print(filled_tensor)
+            ttnn.Tensor([[[[7.0,  7.0],
+                            [7.0,  7.0]]]], shape=Shape([2, 2]), dtype=DataType::BFLOAT16, layout=Layout::ROW_MAJOR)
+        )doc",
+        ttnn::moreh_full.base_name());
+
+    using FullType = decltype(ttnn::moreh_full);
+    bind_registered_operation(
+        module,
+        ttnn::moreh_full,
+        doc,
+        ttnn::pybind_arguments_t{
+            py::arg("shape"),
+            py::arg("fill_value"),
+            py::arg("any"),
```

Comment:
what is any? output tensor? lets give it a proper name

Reply:
I've updated the doc for better clarity.